setup.py

```diff
@@ -55,11 +55,11 @@ def unique_flatten_dict(d):
 }
 
 base_extras_heavy = {
-  'umap-learn': ['umap-learn', 'dirty-cat==0.2.0', 'scikit-learn>=1.0'],
+  'umap-learn': ['umap-learn','skrub', 'scikit-learn'],
```

Comment:
Do you think we need to pin `skrub`?
Its dependencies (including `dirty-cat`) are not pinned:
https://github.com/skrub-data/skrub/blob/main/.binder/requirements.txt